pkg/cluster/cluster.go

```diff
@@ -124,35 +124,40 @@ func new(config Config, s *spec.ClusterSpec, stopC <-chan struct{}, wg *sync.Wai
 func (c *Cluster) prepareBackupAndRestore() error {
 	backup := c.spec.Backup
 	restore := c.spec.Restore
+
+	if backup == nil {
+		if restore != nil {
+			return errors.New("restore policy set but backup policy not. This is not allowed. Cluster to be dead.")
+		} else {
+			return nil
+		}
+	}
+
 	switch {
-	case backup == nil && restore != nil:
-		return errors.New("restore policy set but backup policy not. This is not allowed. Cluster to be made dead.")
-	case backup != nil && restore == nil:
+	case restore == nil:
 		err := k8sutil.CreateAndWaitPVC(c.KubeCli, c.Name, c.Namespace, c.PVProvisioner, backup.VolumeSizeInMB)
 		if err != nil {
 			return err
 		}
-		fallthrough
-	case backup != nil && restore != nil:
-		if restore != nil && restore.BackupClusterName != c.Name {
-			err := k8sutil.CreateAndWaitPVC(c.KubeCli, c.Name, c.Namespace, c.PVProvisioner, backup.VolumeSizeInMB)
-			if err != nil {
-				return err
-			}
-			err = k8sutil.CopyVolume(c.KubeCli, restore.BackupClusterName, c.Name, c.Namespace)
-			if err != nil {
-				return err
-			}
+	case restore != nil:
+		c.logger.Infof("will restore from existing backup if no etcd member running")
```

Comment:
well. can you combine the two pr then? i do not want to see half sentence here. or can you simply remove this logging line?